setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="Open edX",
-    version="0.9",
+    version="0.10",
```

Comment:
Whoa really? Is platform version not a branding issue? What's the intent here?

Reply:
@nasthagiri mentioned that this was necessary when adding new plugins.

Reply:
Yes, that's right.  The version bump will trigger a re-install of the setup file, which is needed to detect new plugins.  No, it doesn't imply an up-branding of the platform (at least not now).